### ISSUE TITLE
Alex's Bald Eagle causing very bad lag.

### ISSUE THREAD

**Issue by Splutty** [NillerMedDild/Enigmatica6#2453]
**Modpack version**
0.50 Alpha

**Describe the bug**
Alex's Bald Eagle causes extreme lag. https://spark.lucko.me/tMt9vXKTPU

No actual errors in any logs.

**Logs**
https://spark.lucko.me/tMt9vXKTPU

**Comment by NillerMedDild** [comment]
Thank you for the report, I will investigate the issue.

**Comment by landretk** [comment]
Bald eagle crashed my server due to exceeding tick time. I have Sodium(includes lithium) on my server though. I was feeding a cat fish and my friend was fishing at the time(probably where the eagle was encountered). But it was stuck somewhere in the entity collision functions like Splutty is seeing.

https://pastebin.com/1rwQUxza

I have seen other bald eagles in the wild without the server lagging. I'm guessing it's during their catching fish out of water routine.

**Comment by Splutty** [comment]
For a workaround, this works to remove all of them:
/cu entities remove byType alexmobs:baldeagle

**Comment by landretk** [comment]
Yeah I cleared mine up with just a /kill @e[type=alexsmobs:bald_eagle]
